components/style/stylesheets.rs

```diff
@@ -99,6 +103,64 @@ pub struct Namespaces {
     pub prefixes: FnvHashMap<Prefix , Namespace>,
 }
 
+/// Like gecko_bindings::structs::MallocSizeOf, but without the Option<> wrapper. Note that
+/// functions of this type should not be called via do_malloc_size_of(), rather than directly.
+#[cfg(feature = "gecko")]
+pub type MallocSizeOfFn = unsafe extern "C" fn(ptr: *const c_void) -> usize;
+
+/// Call malloc_size_of on ptr, first checking that the allocation isn't empty.
+#[cfg(feature = "gecko")]
```

Comment:
Is there a reason this all needs to be `cfg`'d out? If we can compile it in servo mode too it'd be better IMO (less differences among builds).